napari/_qt/dialogs/shimmed_plugin_dialog.py

```diff
@@ -0,0 +1,83 @@
+from qtpy.QtWidgets import (
+    QCheckBox,
+    QDialog,
+    QHBoxLayout,
+    QLabel,
+    QPushButton,
+    QVBoxLayout,
+    QWidget,
+)
+
+from napari.settings import get_settings
+from napari.settings._constants import PluginShimWarningLevel
+from napari.utils.translations import trans
+
+
+class ShimmedPluginDialog(QDialog):
+    def __init__(self, parent: QWidget, plugins: set[str]) -> None:
+        super().__init__(parent)
+        self.setWindowTitle('Installed Plugin Warning')
+        okay_btn = QPushButton(trans._('Okay'))
+        icon_label = QWidget()
+        icon_label.setObjectName('warning_icon_element')
+        self.plugin_info_text = """
+The following plugins use napari's old plugin engine. These plugins have been automatically converted to the new plugin engine, npe2:
+        """
+        self.plugins = plugins
+        self.plugin_text = '\n'.join(self.plugins)
+        self.info_text = """
+While the conversion should work for these plugins, some plugin functionality, such as code designed to run on import, may not work as expected.
+
+If you notice an error in any of these plugins, drop back to using the old plugin engine by turning off the 'Use npe2 adaptor' setting in the plugin preferences.
```

Comment:
```suggestion
If you see an error with these plugins, disable the 'Use npe2 adaptor' setting in plugin preferences. This will drop back napari to use the deprecated plugin engine.
```